lib/tasks/backend.rake

```diff
@@ -0,0 +1,23 @@
+namespace :backend do
+  desc 'Updates backend_url for a given backend'
+  task modify_url: [:environment] do
+    unless ENV['BACKEND_ID'] && ENV['BACKEND_URL']
```

Comment:
I think it'll work with environment variables anyway but will take a look at that, thanks!